infrastructure/variables.tf

```diff
@@ -33,5 +33,6 @@ variable "subscription" {
 }
 
 variable "testing_support" {
+  default = ""
```

Comment:
perhaps `false`?

Reply:
seems fine with false - making the change and merging